ecs-logs-collector.sh

```diff
@@ -496,6 +499,59 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/docker-sysconfig
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+  fi
+
+ try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker-storage "${info_system}"/docker/docker-sysconfig-storage
+    ok
+  else
+    info "/etc/sysconfig/docker-storage not found"
+  fi
+}
+
+
+get_docker_daemon_json(){
+  try "Gathering Docker daemon.json"
+
+  if [ -e /etc/docker/daemon.json ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/docker/daemon.json "${info_system}"/docker/daemon.json
+    ok
+  else
+    info "/etc/docker/daemon.json not found"
+  fi
+}
+
+get_docker_systemd_config(){
+  try "Gather Docker Daemon Service file"
+  
+  # Checking Rhel based distros
+  if [ -e /usr/lib/systemd/system/docker.service ]; then
+    mkdir -p "${info_system}"/docker
+    cp /usr/lib/systemd/system/docker.service "${info_system}"/docker/docker.service
```

Comment:
This isn't the only locations that units can be configured from (this is typically just the distribution's packaged units fwiw). Additional configuration can come from runtime edits or other drop in units. Please use `systemctl cat docker.service` to capture the effective unit according to systemd.

Reply:
Changed to `systemctl cat docker.service`